win32/src/winapi/gdi32/dc.rs

```diff
@@ -134,7 +136,8 @@ pub fn GetDeviceCaps(
         GetDeviceCapsArg::NUMCOLORS => -1i32 as u32, // true color
         GetDeviceCapsArg::HORZRES => 640,
         GetDeviceCapsArg::VERTRES => 480,
-        _ => unimplemented!(),
+        GetDeviceCapsArg::RASTERCAPS => 0x100, // RC_PALETTE
```

Comment:
Does this need to depend on the underlying device?  I think this is making this function say that all our DCs support palettes, when I think in practice none (?) of them do.

Reply:
I'm not 100% sure here, but what I assumed when I wrote it was that the palettes would be handled by some code that I will write soon. But with that said, I'm not really sure what these paletts do, and how they are used at the moment 😅 

Maybe it's better to hold of on this one until I have gotten more of it to work...